site/content/docs/5.0/components/popovers.md

```diff
@@ -269,12 +269,25 @@ Note that for security reasons the `sanitize`, `sanitizeFn`, and `allowList` opt
       <td>null | function</td>
       <td><code>null</code></td>
       <td>Here you can supply your own sanitize function. This can be useful if you prefer to use a dedicated library to perform sanitization.</td>
+      <tr>
```

Comment:
There: you're opening a `<tr>` without closing the previous one…

Reply:
Thanks ! 🙏 Correcting right now